### ISSUE TITLE
Sensors plugin error during initialization in CodeSpace

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#2772]
Inside Codespace (Github) when Glances is ran, the following error occurs:

```
2024-05-15 08:52:10,387 -- CRITICAL -- Error while initializing the sensors plugin ('type' object is not subscriptable)
2024-05-15 08:52:10,391 -- ERROR -- Traceback (most recent call last):
  File "/workspaces/glances/glances/stats.py", line 104, in _load_plugin
    plugin = import_module('glances.plugins.' + plugin_path)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/workspaces/glances/glances/plugins/sensors/__init__.py", line 320, in <module>
    class GlancesGrabSensors(object):
  File "/workspaces/glances/glances/plugins/sensors/__init__.py", line 350, in GlancesGrabSensors
    def update(self) -> list[dict]:
TypeError: 'type' object is not subscriptable
```

**Comment by nicolargo** [comment]
cc: @RazCrimson

**Comment by nicolargo** [comment]
@RazCrimson i confirm that the bug no longer exist in released 4.04 but it it not corrected in develop branch. Merged to be done.
